src/content/troubleshooting/_index.en.md

```diff
@@ -0,0 +1,131 @@
++++
+title = "Troubleshooting Guide"
+date = 2020-05-04T19:01:14+05:30
+weight = 5
+pre = "<b>4. </b>"
++++
+
+## Overview
+
+You have followed steps in [Deployment](../deployment) but something has gone wrong. You're not sure what and how to fix it, or what information to collect to raise an issue. Welcome to the Submariner troubleshooting guide where we will help you get your deployment working again.
+
+Basic familiarity with the Submariner components and architecture will be helpful when troubleshooting so please review the [Architecture](../architecture) section.
+
+The guide has been broken into different sections for easy navigation.
+
+### Pre-requisite
+Before we begin troubleshooting, run `subctl version` to obtain which version of the Submariner components you are running.
+
+Run `kubectl get services | grep <service-name>` to get information about the service you're trying to access. This will provide you with the Service *Name*, *Namespace* and *ServiceIP*. If **Globalnet** is enabled, you will also need the *globalIp* of the service by running
+
+``` kubectl get service <service-name> -o jsonpath='{.metadata.annotations.submariner\.io/globalIp}' ```
+
+<!---
+### Deployment Issues
+This section will contain information about common deployment issues you can run into.
+
+#### TBD
+
+### Connectivity Issues
+Deployment went through successfully but services/pods on one cluster are unable to connect to services on another clusters. This can be due to multuple factors - IP Sec tunnels, IP Table rules, Gateway pods, Routeagent etc.
+
+#### IPSec tunnel not created between clusters
+TBD
+
+#### IPSEc tunnel is not up between clusters
+TBD
+
+#### None of pods/services able to connect to remote service
+TBD
+##### Without Globalnet
+TBD
+##### With Globalnet
+TBD
+
+#### Pods on non-gateway nodes not able to connect to remote service
+TBD
+##### Without Globalnet
+TBD
+##### With Globalnet
+TBD
+
+-->
+
+### Service Discovery Issues
+If you are able to connect to remote service by using ServiceIP or globalIp, but not by service name, it is a Service Discovery Issue.
+
+#### Service Discovery not working
+This is good time to familiarize yourself with [ServiceDiscovery Architecture](../architecture/service-discovery/) if you haven't already.
+
+##### Check CoreDNS Configuration
+Submariner configures CoreDNS deployment to enable `lighthouse` plugin. Since CoreDNS is the first point of entry for any DNS queries, that is where we will start first. 
+
+First we check if we are running the correct image. We need CoreDNS image with ligthouse enabled.
+
+```kubectl -n kube-system kube-dns describe deployment```
```

Comment:
In OpenShift, I dont see the kube-dns deployment. Can you also include a note about equivalent command for OCP?

Reply:
Made a generic note to cover all non-vanilla deployments.

Reply:
I tried the command in OCP and it does not show any dns deployments in openshift-dns namespace.

[sgaddam@localhost aws-sgaddam]$ kubectl -n openshift-dns get deployment
No resources found.
[sgaddam@localhost aws-sgaddam]$